cytoscape-navigator.js

```diff
@@ -889,6 +889,8 @@
       $img.style['position'] = 'absolute';
       $img.style['left'] = translate.x + 'px';
       $img.style['top'] = translate.y + 'px';
+      $img.style['width'] = '100%';
+      $img.style['height'] = '100%';
```

Comment:
`100%` only works if the parent DOM element has specific `position` style values.  It would be better to use a pixel value like `viewW`.

Reply:
what do you mean by `viewW`? Is it `vw` https://www.w3schools.com/CSSref/tryit.asp?filename=trycss_unit_vw ?

Reply:
I think there are fields for the the view width and the overall width (in pixels) stored under `this`.  We need to use `px` values rather than relative `%` units.